tests/foreman/cli/test_activationkey.py

```diff
@@ -558,11 +558,12 @@ def test_positive_usage_limit(self):
                 vm1.install_katello_ca()
                 result = vm1.register_contenthost(
                     self.org['label'], new_ak['name'])
-                self.assertEqual(result.return_code, 0)
+                self.assertTrue(vm1._subscribed)
```

Comment:
@rplevka , you are accessing a private attr instead of doing and as you are refactoring  VirtualMachine you can add a property in VirtualMachine class:

```python
class VirtualMachine(object):
    ...
    @property
    def subscribed(self):
        return self._subscribed

```